InfluxPublisher/README.md

```diff
@@ -6,15 +6,15 @@ This folder contains Lingua Franca reactors that publish to InfluxDB via a socke
 
 1. Compile the LF example applications:
 
-    ```bash
-    cd c && lfc src/InfluxPublisher.lf
-    cd py && lfc src/InfluxPublisher.lf
-    ```
+   ```bash
+   cd c && lfc src/InfluxPublisher.lf; cd ..
+   cd py && lfc src/InfluxPublisher.lf; cd ..
+   ```
 
 1. Start Xronos Dashboard and LF example applications:
 
-    ```bash
-    docker compose up
-    ```
+   ```bash
+   docker compose up
+   ```
 
-    Open Grafana on `http://0.0.0.0:3000` and open the dashboard "Telegraf Test Publishers".
+   Open Grafana on `http://localhost:3000` and open the dashboard "Telegraf Test Publishers".
```

Comment:
Curious why this change?

Reply:
See https://github.com/xronos-inc/xronos-dashboard/pull/3#discussion_r1698997833. That's all I know.